src/modules/endpoint/endpoints/__tests__/events.spec.js

```diff
@@ -39,31 +60,6 @@ describe('events', () => {
     )
   })
 
-  it('should throw if the city to map the url are missing', () => {
-    expect(() => events.mapParamsToUrl({})).toThrowErrorMatchingSnapshot()
-  })
-
-  it('should throw if the language to map the url are missing', () => {
-    expect(() => events.mapParamsToUrl({city: 'city'})).toThrowErrorMatchingSnapshot()
-  })
-
-  const toEventModel = json => {
```

Comment:
@klinzo found this function while fixing thix test. I think this test is quite old, but we should not duplicate intentional funtionality for tests.